tests/caasp/stack_conformance.pm

```diff
@@ -21,29 +21,48 @@ use testapi;
 use caasp 'script_retry';
 use version_utils 'is_caasp';
 
+my $config_json = <<'EOF';
+{
+  "plugins": [ { "name": "e2e" } ]
+}
+EOF
+
 sub run {
-    switch_to 'xterm';
+    # The repository for the sonobuoy package
+    my $repo = "https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Tumbleweed/devel:kubic.repo";
```

Comment:
?  isn't better use `get_var('SOME_VAR_WITH_SONO_REPO')`

Reply:
thanks!!! Indeed it makes much more sense!